include/sound/sof/info.h

```diff
@@ -25,6 +25,7 @@
 #define SOF_IPC_INFO_LOCKS		BIT(1)
 #define SOF_IPC_INFO_LOCKSV		BIT(2)
 #define SOF_IPC_INFO_GDB		BIT(3)
+#define SOF_IPC_INFO_D3_PERSISTENT	BIT(4)
```

Comment:
I am not good at this naming stuff, since it has gone with this on the FW side as a FW_READY flag, let's just align to it?

For the ABI version, can we use 3.19 for both updates? It is a little bit screwed as the FW side has taken this D3_PERSISTENT as 3.19 and driver side took SSP changes as 3.19.
To fix this, as before can we use one minor version bump for several updates in a bunch? @lgirdwood FYI.

Reply:
@plbossart @keyonjie we can use 3.19 for both 3.19 already went out with a release (and I thin that's not the case, the tagged v1.8 has ABI3.18.1), so 3.19 is for SOF v1.9.

Reply:
@kv2019i I don't think 3.19 is out, I bumped the number in my SSP changes and it's not merged yet.